schemas/com.google.analytics.ecommerce/item/jsonschema/1-0-0

```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Google Analytics Ecommerce transaction item",
+	"self": {
+		"vendor": "com.google.analytics.ecommerce",
+		"name": "item",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string",
+			"maxLength": 500
+		},
+		"price": {
+			"type": "number",
+			"multipleOf": 0.01
+		},
+		"quantity": {
+			"type": "integer"
+		},
+		"code": {
+			"type": "string",
+			"maxLength": 500
+		},
+		"category": {
+			"type": "string",
+			"maxLength": 500
+		},
+		"currency": {
+			"type": "string",
+			"maxLength": 16
```

Comment:
Per https://support.google.com/analytics/answer/6205902#supported-currencies and "The local currency must be specified in the ISO 4217 standard" [1] I think we can safely make all currency fields minLength 3, maxLength 3...

[1] https://developers.google.com/analytics/devguides/collection/analyticsjs/ecommerce#multicurrency